pctiler/pctiler/colormaps/viirs.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+
+from rio_tiler.types import ColorMapType
+
+viirs_colormaps: Dict[str, ColorMapType] = {
```

Comment:
This needs to be imported into the module `__init__` to get it registered in the service. I think we need to improve how this happens, but for now it's just a manual step.

https://github.com/microsoft/planetary-computer-apis/blob/93ee00039b7da8646b6f13b2b523b9b6acdaad2a/pctiler/pctiler/colormaps/__init__.py#L24-L33

Reply:
Thanks for the feedback, Matt! viirs_colormaps is import into __init__.py. I'll make a note of this step on our wiki.